app/models/manageiq/providers/openstack/manager_mixin.rb

```diff
@@ -115,7 +115,7 @@ def verify_credentials(args)
     end
 
     def raw_connect(password, params, service = "Compute")
-      params[:proxy] = openstack_proxy.to_s if openstack_proxy
+      params[:proxy] = openstack_proxy
```

Comment:
Travis failures look related, I think you might still need the `if openstack_proxy`

Reply:
Ah, so explicit nils are causing problems? I was afraid of that.

Reply:
Just guessing because that looks like the major difference between the commits

Reply:
Nice looks like that did the trick :+1: